src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -675,6 +675,27 @@ public void execute(Boolean succeeded)
         }
      );
    }
+
+   public void withKeyring(final Command command)
+   {
+     withDependencies(
+        "Preparing Keyring",
+        "Using keyring", 
+        new Dependency[] {
+           Dependency.cranPackage("keyring", "1.0.0", true)
```

Comment:
Can you update the admin guide to include this package?

https://github.com/rstudio/rstudio-pro/blob/6b14277b5aabd8de1b7225291f1cf3099c56f582/docs/server/index.Rmd#L184